kubeadm/scripts/PrepareNode.ps1

```diff
@@ -105,24 +105,26 @@ nssm set kubelet DependOnService containerd
 
 New-NetFirewallRule -Name kubelet -DisplayName 'kubelet' -Enabled True -Direction Inbound -Protocol TCP -Action Allow -LocalPort 10250
 
+$gitpath='https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master'
+
 Write-Output "Please remember that after you have joined the node to the cluster, that you have to apply the cni daemonset/service and the kube-proxy"
 Write-Output "Also remember that for kube-proxy you have to change the its version from the name of the image in the kube-proxy.yml to that of your kubernetes version `n"
 # rancher commands
 Write-Output "In case you use rancher, use the following commands:"
 Write-Output "For Windows you can use the following command: "
-Write-Output "(Get-Content `"$(git rev-parse --show-toplevel)/kubeadm/kube-proxy/kube-proxy.yml`") -Replace 'VERSION', '$KubernetesVersion' | Set-Content `"$(git rev-parse --show-toplevel)/kubeadm/kube-proxy/kube-proxy.yml`" `n"
+Write-Output "(Get-Content `"$gitpath/kubeadm/kube-proxy/kube-proxy.yml`") -Replace 'VERSION', '$KubernetesVersion' | Set-Content `"$gitpath/kubeadm/kube-proxy/kube-proxy.yml`" `n"
```

Comment:
is this right?

```
 Set-Content `"$gitpath/kubeadm/kube-
```

I would have thought this would be local path

Reply:
Yes, I misunderstood the original intention here I think. I think this part of the script assumes that you git cloned the whole repo down. 

However the guide tells you to download this script with curl and run it. So in that case it will not run from within a cloned repo and local paths will not work. So for that reason I think the commands to apply the manifests for kubeproxy should refer to the online manifests by using http urls.

However then of course the `Get-Content` needs to be changed to `curl` commands. I'll do that and see if it makes more sense.

Reply:
I ended up first downloading the manifest and then patching the local file with the version. Do you think this will work?